plaso/cli/status_view.py

```diff
@@ -56,6 +58,12 @@ def __init__(self, output_writer, tool_name):
     self._storage_file_path = None
     self._tool_name = tool_name
 
+    if win32console:
+      kernel32 = ctypes.windll.kernel32
+      stdout_handle = kernel32.GetStdHandle(-11)
```

Comment:
Please extract these values (-11, 7) to class constants.

Reply:
done